modules/org-admin-account/versions.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = ">= 0.15"
+
+  required_providers {
+    aws = {
+      source                = "hashicorp/aws"
+      version               = ">= 3.0"
+      configuration_aliases = [aws.guardduty_administrator]
```

Comment:
Can be removed, since we aren't actually using multiple providers in this module.

Reply:
Removed configuration_aliases from versions.tf.  Also removed the providers block from the module /tests/create_guardduty_org_admin_account/main.tf since it is just using a non-aliased provider.